src/main/scala/com/microsoft/azure/iot/iothubreact/scaladsl/IoTHub.scala

```diff
@@ -8,15 +8,16 @@ import akka.stream._
 import akka.stream.scaladsl._
 import akka.{Done, NotUsed}
 import com.microsoft.azure.iot.iothubreact._
-import com.microsoft.azure.iot.iothubreact.checkpointing.{Configuration ⇒ CPConfiguration}
+import com.microsoft.azure.iot.iothubreact.checkpointing.{Configuration => CPConfiguration}
 import com.microsoft.azure.iot.iothubreact.sinks.{DevicePropertiesSink, MessageToDeviceSink, MethodOnDeviceSink}
+import com.typesafe.config.ConfigFactory
 
 import scala.concurrent.Future
 import scala.language.postfixOps
 
 /** Provides a streaming source to retrieve messages from Azure IoT Hub
   */
-case class IoTHub() extends Logger {
+case class IoTHub(implicit val cfg: CPConfiguration = new CPConfiguration()) extends Logger {
```

Comment:
ditto: ICPConfiguration, without default (see my other comment, same applied to other ctors)